Makefile

```diff
@@ -33,7 +33,7 @@ export GOPROXY
 export GO111MODULE=on
 
 # Kubebuilder.
-export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.20.2
+export KUBEBUILDER_ENVTEST_KUBERNETES_VERSION ?= 1.28.0
```

Comment:
I think the test failure is the same as #3420.

Reply:
That's odd. The same change seems to have worked in #4464.

I'll revert this part and see if that gets things passing.